docs/source/AdministratorGuide/Resources/proxyprovider.rst

```diff
@@ -0,0 +1,85 @@
+.. _resourcesProxyProvider:
+
+==============
+ProxyProvider
+==============
+
+This resource type provide interface to obtain proxy certificate using user indentifier. The following proxy providers are represented here: ``DIRACCA``, ``PUSP``.
+
+
+DIRACCA proxy provider
+----------------------
+
+ProxyProvider implementation for the proxy generation using local CA credentials. This class is a simple CA, it's main purpose is to generate a simple proxy on the fly for DIRAC users who do not have any certificate register. To use it need to have CA certificate and key localy. Required parameters in the DIRAC configuration for its implementation::
+
+    Resources
+    {
+      ProxyProviders
+      {
+        MY_DIRAC_CA
+        {
+          ProviderType = DIRACCA
+          CertFile = /opt/dirac/pro/etc/grid-security/MY_DIRAC_CA.pem
+          KeyFile = /opt/dirac/pro/etc/grid-security/MY_DIRAC_CA.key.pem
+          Match = O, OU
+          Supplied = CN
+          Optional = emailAddress
+          DNOrder = O, OU, CN, emailAddress
+        }
+      }
+    }
+
+Configuration options are:
+
+* ``ProviderType``: main option, to show which proxy provider type you want to register. In the case of the DIRACCA proxy provider you must type ``DIRACCA``.
+* ``CertFile``: the path to the CA certificate.
+* ``KeyFile``: the path to the CA key.
+* ``CAConfigFile``: the path to the openssl configuration file. This is optional and not recomended to use. But if you choose to use this option, it is recommended to use a relatively simple configuration.
+* ``Match``: the distinguished name fields that must contain the exact same contents as that field in the CA's DN.
+* ``Supplied``: the distinguished name fields list that must be present.
+* ``Optional``: the distinguished name fields list that are allowed, but not required.
+* ``DNOrder``: order of the distinguished name fields in a created user certificate.
```

Comment:
Are all these parameters needed, or they are anyway the defaults?

Reply:
good point,
Here are two ways to set up this type of proxy provider:
1. all the necessary parameters set in CS:
    all of these options are required, but each has a default value if it will not set in CS.
2. set in CS only the existing path of the openssl configuration file `CAConfigFile` and `DNOrder` is optional here:
    in this case all parameters will be read from this file.

So I'll add this to the description.